### PR TITLE
chore(description): update the description meta tag

### PR DESCRIPTION
**Pull request by robinjoseph08**
(๑•̌.•̑๑)ˀ̣ˀ̣

no idea

### notes

- i know this isn't super important right now, but i cant stop thinking about it so ima do it anyway
- gots some infos from [here](https://support.google.com/gsa/answer/6329200#2ca1bec8-9cb6-4e70-a959-8993bc3d4a61)
- if you think any copy should change, lmk

### CODE REVIEW

**Comment by robinjoseph08** [comment]
This is what I wanna change:

screen shot 2016-04-06 at 9 39 13 pm

File: public/index.html [hunk 1]
```diff
@@ -23,13 +23,13 @@
     <meta name="msapplication-TileColor" content="#ffffff">
     <meta name="msapplication-TileImage" content="/favicon/ms-icon-144x144.png">
     <meta name="theme-color" content="#ffffff">
-    <meta name="description" content="A website to track your completion of a Living Pokédex">
-    <meta name="keywords" content="pokemon,pokedex,tracker,checklist,livingdex">
+    <meta name="description" content="A tool for tracking your Living Dex progress. Easily toggle between and track your captured Pokémon, find the locations of those left to be captured, and share a public link with others to see how you can help each other out.">
+    <meta name="keywords" content="pokemon,pokedex,track,tracker,checklist,livingdex,progress,completion">
```
Comment (ami) [inline public/index.html]: can you add "living dex" and "pokedex tracker"? or are two word phrases not allowed

File: public/index.html [hunk 2]
```diff
@@ -23,13 +23,13 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0">
     <meta name="og:url" content="https://pokedextracker.com/">
-    <meta name="og:image" content="http://pokedextracker.com/og.png">
+    <meta name="og:image" content="https://pokedextracker.com/og.png">
```
Comment (ami) [inline public/index.html]: does this mean i need to make one of these? lol

File: public/index.html (hunk 1 above)
Comment (robinjoseph08) [inline public/index.html]: They are! I'll add it!

File: public/index.html (hunk 2 above)
Comment (robinjoseph08) [inline public/index.html]: 😁

Comment (ami) [inline public/index.html]: sigh. i'll make an issue for that

**Comment by robinjoseph08** [comment]
@ami updated!

File: public/index.html [hunk 3]
```diff
@@ -23,13 +23,13 @@
     <meta name="msapplication-TileColor" content="#ffffff">
     <meta name="msapplication-TileImage" content="/favicon/ms-icon-144x144.png">
     <meta name="theme-color" content="#ffffff">
-    <meta name="description" content="A website to track your completion of a Living Pokédex">
-    <meta name="keywords" content="pokemon,pokedex,tracker,checklist,livingdex">
+    <meta name="description" content="A tool for tracking your Living Dex progress. Easily toggle between and track your captured Pokémon, find the locations of those left to be captured, and share a public link with others to see how you can help each other out.">
+    <meta name="keywords" content="pokemon,pokedex,track,tracker,checklist,livingdex,progress,completion, pokedex tracker,living dex">
```
Comment (ami) [inline public/index.html]: extra space?

**Comment by ami** [comment]
lgtm

File: public/index.html (hunk 3 above)
Comment (robinjoseph08) [inline public/index.html]: asdf

**Comment by robinjoseph08** [comment]
@ami updated again...

**Pull request merged**